src/pages/auth/auth.css

```diff
@@ -0,0 +1,85 @@
+/* subscribe modal */
+.subscribe-section-overlay,
+.login-section-overlay,
+.signup-section-overlay,
+.signin-section-overlay {
+  background: var(--light-black);
+  backdrop-filter: blur(2px);
+  width: 100vw;
+  height: 100vh;
+  z-index: 99999;
```

Comment:
changes: use a minimum number for z-index and also use CSS variable for z-index.